snakefile.py

```diff
@@ -147,22 +159,34 @@ def tool(name):
             for name in generated:
                 print("  - {}".format(name))
 
+# function to pass read files to trim/filter/qc improvement
```

Comment:
This is nitpicking, but I'd prefer if the comment was a little more accurate: Return an array of one or two read file names for the given sample name.